<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpatialForm</class>
 <widget class="QWidget" name="SpatialForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>238</width>
    <height>116</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>空间分析</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/img/board_background.jpg);</string>
  </property>
  <widget class="QPushButton" name="pointMeasureBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pointMeasureBtn
{
	
	border: 2px solid #9ea1a4;
	border-radius: 8px;
	image: url(:/MyProjectUI/Resources/img/pointMeasure.png);
	
	
}
QPushButton#pointMeasureBtn:hover	
{
	background-color: rgb(188, 255, 250);
	border: 2px solid #9ea1a4;
	border-radius: 8px;
	image: url(:/MyProjectUI/Resources/img/pointMeasure.png);
	
	
}
QPushButton#pointMeasureBtn:pressed
{
	background-color: rgb(0, 255, 255);
	border: 2px solid #9ea1a4;
	border-radius: 8px;
	image: url(:/MyProjectUI/Resources/img/pointMeasure.png);
	
	
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="distanceMeasureBtn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#distanceMeasureBtn
{
	border: 2px solid #9ea1a4;
	border-radius: 8px;	
    image: url(:/MyProjectUI/Resources/img/distaceMeasure.png)
}
QPushButton#distanceMeasureBtn:hover
{
	background-color: rgb(188, 255, 250);
	border: 2px solid #9ea1a4;
	border-radius: 8px;	
    image: url(:/MyProjectUI/Resources/img/distaceMeasure.png)
}
QPushButton#distanceMeasureBtn:pressed
{
	background-color: rgb(0, 255, 255);
	border: 2px solid #9ea1a4;
	border-radius: 8px;	
    image: url(:/MyProjectUI/Resources/img/distaceMeasure.png)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clearMeasureBtn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#clearMeasureBtn
{
	border: 2px solid #9ea1a4;
	border-radius: 8px;
	image: url(:/MyProjectUI/Resources/img/clear.png);
}
QPushButton#clearMeasureBtn:hover	
{
	background-color: rgb(188, 255, 250);
	border: 2px solid #9ea1a4;
	border-radius: 8px;
	image: url(:/MyProjectUI/Resources/img/clear.png);
}
QPushButton#clearMeasureBtn:pressed
{
	background-color: rgb(0, 255, 255);
	border: 2px solid #9ea1a4;
	border-radius: 8px;
	image: url(:/MyProjectUI/Resources/img/clear.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>点位量测</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>距离量测</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>清除量测</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../../MyProjectUI/myprojectui.qrc"/>
  <include location="img/spa.qrc"/>
 </resources>
 <connections/>
</ui>
